### DISCUSSION TITLE
🦊 v3.7.x

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris#5209] [Releases]
<div align="center">

| Manager | Buddy | Feature Branch | Project |
| :---: | :---: | :---: | :---: |
| @pp-mo | @HGWright (@bjlittle) | - | [![GH Project](https://img.shields.io/badge/project-%F0%9F%A6%8A%20v3.7.0-yellow?logo=github&logoColor=lightgrey)](https://github.com/orgs/SciTools/projects/18) |

</div>

<div align="center">

| Minor Release | Status | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.7.0rc0` | [![status](https://img.shields.io/badge/v3.7.0rc0-released-green)](https://github.com/SciTools/iris/discussions/5209) | `10-08-2023` | - | - | - | - |
| `v3.7.0` | [![status](https://img.shields.io/badge/v3.7.0-released-green)](https://github.com/SciTools/iris/discussions/5209) | `31-08-2023` | - | - | - | - |
| `v3.7.1` | [![status](https://img.shields.io/badge/v3.7.1-released-green)](https://github.com/SciTools/iris/discussions/5209) | `04-03-2023` | [RTD](https://scitools-iris.readthedocs.io/en/v3.7.1/whatsnew/index.html#v3-7-1-04-mar-2024) | [v3.7.1](https://github.com/SciTools/iris/releases/tag/v3.7.1) | [conda-forge](https://anaconda.org/conda-forge/iris?version=3.7.1&channel=main) | [PyPI](https://pypi.org/project/scitools-iris/3.7.1) |

</div>

**Comment by trexfeathers** [comment]
@pp-mo @bjlittle please note the minor conda-forge recipe changes next time we do a release candidate:

conda-forge/iris-feedstock#96

**Comment by pp-mo** [comment]
Sprint Project 
https://github.com/orgs/SciTools/projects/18

**Comment by pp-mo** [comment]
Release candidate produced, and now out on PyPI/conda-forge.

Release page (includes PyPI + conda-forge links) :
https://github.com/SciTools/iris/releases/tag/v3.7.0rc0

**Comment by bjlittle** [comment]
@pp-mo Could you update the release details (above) for the 3.7.x release candidate and release please :+1:

**Comment by pp-mo** [comment]
Now produced the final release.
Discussions should now be directed at the new [release discussion page](https://github.com/SciTools/iris/discussions/5452).

( Release page :  https://github.com/SciTools/iris/releases/tag/v3.7.0
Includes links for releases on [PyPI](https://pypi.org/project/scitools-iris/3.7.0) and [conda-forge](https://anaconda.org/conda-forge/iris?version=3.7.0&channel=main).
)
